pandarallel/core.py

```diff
@@ -239,7 +240,7 @@ def closure(
 
         show_progress_bars = progress_bars_type != ProgressBarsType.No
 
-        progress_bars = get_progress_bars(progresses_length, show_progress_bars)
+        progress_bars = get_progress_bars(progresses_length, show_progress_bars, max_progress_bars)
```

Comment:
Check line length -- I think this repo (generally) follows the PEP8 rule of max 79 character lines. @nalepae do we have an auto-formatter that we use?